### PR TITLE
[docs] Add docs for notification center

### PR DESCRIPTION
**Pull request by aabidsofi19**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 5e27a562387ea7f9bbc1c27ec57674b0c21bf008 at: https://6544104e8452f26af461ce4a--meshery-docs-preview.netlify.app

File: docs/pages/guides/events-management.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Summary
+
+Meshery's Notification Center simplifies the task of managing events in your Kubernetes and Cloud-Native environments. This robust feature offers a central hub to view, search, and filter events, making it effortless to identify and address critical incidents. Each event comes with detailed descriptions and additional data for efficient troubleshooting.
+
+You can easily collaborate with stakeholders by sharing event data, enhancing communication and issue resolution. What's more, Meshery enables you to track event statuses, ensuring no crucial event goes unnoticed. Plus, events are always persisted, providing a historical record for compliance and trend analysis. With Meshery's Notification Center, you can gain deep insights into your infrastructure, swiftly respond to critical events, and maintain optimal performance for your applications and services. Detailed instructions and best practices can be found in the official documentation and Meshery's user guide.
```
Comment (hamza-mohd) [inline docs/pages/guides/events-management.md:56]:
```suggestion
You can easily collaborate with stakeholders by sharing event data, enhancing communication and issue resolution. What's more, Meshery enables you to track event statuses, ensuring no crucial event goes unnoticed. Plus, events are always persisted, providing a historical record for compliance and trend analysis. With Meshery's Notification Center, you can gain deep insights into your infrastructure, swiftly respond to critical events, and maintain optimal performance for your applications and services. Detailed instructions and best practices can be found in the official documentation and Meshery's user guide.

{% include suggested-reading.html language="en" %}

{% include related-discussions.html tag="meshery" %}
```

Comment (leecalcote) [reply docs/pages/guides/events-management.md:56]: Good one, @hamza-mohd

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
We'll also want to add an entry in `data/toc.yml`

**Pull request merged**